src/Sulu/Bundle/Sales/OrderBundle/DependencyInjection/Configuration.php

```diff
@@ -26,29 +26,63 @@ public function getConfigTreeBuilder()
             ->arrayNode('pdf_templates')
                 ->addDefaultsIfNotSet()
                     ->children()
-                    ->scalarNode('base')->defaultValue('SuluSalesCoreBundle:Pdf:pdf-base.html.twig')->end()
-                    ->scalarNode('header')->defaultValue('SuluSalesCoreBundle:Pdf:pdf-base-header.html.twig')->end()
-                    ->scalarNode('footer')->defaultValue('SuluSalesCoreBundle:Pdf:pdf-base-footer.html.twig')->end()
-                    ->scalarNode('macros')->defaultValue('SuluSalesCoreBundle:Pdf:pdf-macros.html.twig')->end()
+                    ->scalarNode('base')
+                        ->defaultValue('SuluSalesCoreBundle:Pdf:pdf-base.html.twig')
+                    ->end()
+                    ->scalarNode('header')
+                        ->defaultValue('SuluSalesCoreBundle:Pdf:pdf-base-header.html.twig')
+                    ->end()
+                    ->scalarNode('footer')
+                        ->defaultValue('SuluSalesCoreBundle:Pdf:pdf-base-footer.html.twig')
+                    ->end()
+                    ->scalarNode('macros')
+                        ->defaultValue('SuluSalesCoreBundle:Pdf:pdf-macros.html.twig')
+                    ->end()
                     ->scalarNode('confirmation')
                         ->defaultValue('SuluSalesOrderBundle:Pdf:order.confirmation.html.twig')
                     ->end()
+                    ->scalarNode('dynamically')
+                        ->defaultValue('SuluSalesOrderBundle:Pdf:order.dynamically.html.twig')
+                    ->end()
+                    ->scalarNode('dynamically_base')
+                        ->defaultValue('SuluSalesOrderBundle:Pdf:order.confirmation.html.twig')
+                    ->end()
                 ->end()
             ->end()
-            ->scalarNode('pdf_response_type')->defaultValue('inline')->end()
+            ->scalarNode('pdf_response_type')
+                ->defaultValue('inline')
+            ->end()
+            ->scalarNode('pdf_order_confirmation_name_prefix')
+                ->defaultValue('order_confirmation')
+            ->end()
+            ->scalarNode('pdf_order_unsubmitted_name_prefix')
```

Comment:
pdf_order_unsubmitted_name_prefix isnt necessarily a prefix only used for confirmation and dynamic templates. it is the base of the function being able to recieve wheter it should return a name for a order that has been submitted or one that hasn't... If I name it dynamically it is absolutely linked to the dynamicTemplate functionality...